cd/manager/jobmanager/jobManager.go

```diff
@@ -159,24 +174,26 @@ func (m *JobManager) processJobs() {
 			dequeuedJobs = tempSlice
 			// Recheck the length of `dequeuedJobs` in case any job(s) failed preprocessing and got filtered out
 			if len(dequeuedJobs) > 0 {
-				// Check for any force deploy jobs, and only look at the remaining jobs if no deployments were kicked-off.
-				if !m.processForceDeployJobs(dequeuedJobs) {
-					// Decide how to proceed based on the first job from the list
-					if dequeuedJobs[0].Type == manager.JobType_Deploy {
-						if !m.processDeployJobs(dequeuedJobs) {
-							// If no deploy jobs were launched, process pending anchor jobs. We don't want to hold on to
-							// anchor jobs queued behind deploys because tests need anchors to run, and deploys can't run
-							// till tests complete.
-							m.processAnchorJobs(dequeuedJobs)
-						}
-					} else {
-						// Test and anchor jobs can run in parallel
-						m.processTestJobs(dequeuedJobs)
-						m.processAnchorJobs(dequeuedJobs)
-					}
+				// Check for any force deploy jobs, and only look at the remaining jobs if no deployments were kicked
+				// off.
+				if m.processForceDeployJobs(dequeuedJobs) {
+					processAnchorJobs = false
+				} else
+				// Decide how to proceed based on the first job from the list
+				if dequeuedJobs[0].Type == manager.JobType_Deploy {
+					processAnchorJobs = !m.processDeployJobs(dequeuedJobs)
+				} else {
+					m.processTestJobs(dequeuedJobs)
 				}
 			}
 		}
+		// If no deploy jobs were launched, process pending anchor jobs. We don't want to hold on to anchor jobs queued
+		// behind deploys because tests need anchors to run, and deploys can't run till tests complete.
+		//
+		// Test and anchor jobs can run in parallel.
```

Comment:
```suggestion
		// Test and anchor jobs can run in parallel.
```
comment is not relevant to line below. test jobs no longer here

Reply:
I meant to say that the code below it can run regardless of whether test jobs were started above. We only set the new boolean to false if there was a deployment kicked off. I'll reword this because I want to capture this nuance in the comments.